code-challenges.js

```diff
@@ -9,6 +9,19 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 
 
 
+//I couldnt get this one finished up
+
+
+
+const arrayDeleter = (array) => {
+  let arrayDeleter = array.shift()
```

Comment:
well, we got to the "remove first item" step! I think I would've liked to see some comments perhaps on where you got stuck with the shuffling :)